pkg/upgrade/upgrader/helper.go

```diff
@@ -297,6 +297,13 @@ func buildUpgradeTask(kind, name, openebsNamespace string) *apis.UpgradeTask {
 				PoolName: name,
 			},
 		}
+	case "storagePoolClaim":
```

Comment:
I think we should avoid creating a dummy CR in case of SPC.

Reply:
Removed dummy CR creation.